app/assets/stylesheets/sephora_style_guide/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,42 @@
+.breadcrumb {
+  list-style: none;
+  display: flex;
+  font-size: 11px;
+  height: 11px;
+  padding: 0;
+}
+
+.breadcrumb-item {
+  display: inline-table;
```

Comment:
Sorry forgot to mention earlier. The parent is display flex then why do you need `inline-table` here?